torchvision/prototype/transforms/_misc.py

```diff
@@ -225,28 +226,99 @@ def _transform(
         return inpt.transpose(*dims)
 
 
-class RemoveSmallBoundingBoxes(Transform):
-    _transformed_types = (datapoints.BoundingBox, datapoints.Mask, datapoints.Label, datapoints.OneHotLabel)
+class SanitizeBoundingBoxes(Transform):
+    # This removes boxes and their corresponding labels:
+    # - small or degenerate bboxes based on min_size (this includes those where X2 <= X1 or Y2 <= Y1)
+    # - boxes with any coordinate outside the range of the image (negative, or > spatial_size)
+    _transformed_types = datapoints.BoundingBox
 
-    def __init__(self, min_size: float = 1.0) -> None:
+    def __init__(self, min_size: float = 1.0, labels="default") -> None:
```

Comment:
"default" is pretty expressive, but maybe `None` is a good enough sentinel here? The other options are str and callable so that could be an option.

Reply:
That reminds me I didn't go for `None` because we should allow `labels=None`, which means "I don't have labels and I don't care" - that should be allowed right?

Reply:
Fair enough.